rts/lra/test/pom.xml

```diff
@@ -161,6 +161,7 @@
         <!-- the AppServerCoordinatorDeploymentObserver creates the deployment named 'lra-coordinator' and deploys it to the app server,
                      the last part of the path with value 'lra-coordinator' is defined by LRA coordinator JAX-RS endpoint -->
         <lra.coordinator.url>http://${lra.coordinator.host}:${lra.coordinator.port}/lra-coordinator/lra-coordinator</lra.coordinator.url>
+        <lra.coordinator.war.name>lra-coordinator.war</lra.coordinator.war.name>
         <lra.coordinator.xml.filename>lra-coordinator.xml</lra.coordinator.xml.filename>
```

Comment:
What is this file for, it is not referenced or documented anywhere and we don't use it in any of our builds.
Note I am reviewing this line because the last three commits you made are all about the rewrite of our approach to testing LRA.

Reply:
IIRC, `lra.coordinator.war.name` is used by Maven Wildfly Plugin to name the deployed WAR

Reply:
I am referring to line 164.

Reply:
`<lra.coordinator.war.name>lra-coordinator.war</lra.coordinator.war.name>` is used at line [280](https://github.com/jbosstm/narayana/pull/1881/files#diff-89636f088428917c8cfb617b2ba8af009f37ddbc65281cd6d63651c861853a45R280) of the same file. I use a property just in case this name should be used somewhere else

Reply:
I am referring to the line on which I added my comment (and my comment said `Note I am reviewing this line because the last three commits you made are all about the rewrite of our approach to testing LRA. to clarify why I was picking on that line`), namely

``` xml
<lra.coordinator.xml.filename>lra-coordinator.xml</lra.coordinator.xml.filename>
```

Reply:
> ... namely

Thanks for clarifying this.

```<lra.coordinator.xml.filename>lra-coordinator.xml</lra.coordinator.xml.filename>```
basically, the filename indicated here is used by one of the instances of Wildfly, specifically, the one that hosts the LRA coordinator. Both the Maven WildFly Plugin and Arquillian use this file to load the WildFly configuration needed. Another identical file is used for LRA participants (IIRC the filename [`lra-participant.xml`] is defined in the property `lra.participant.xml.filename`)

Reply:
> What is this file for, it is not referenced or documented anywhere and we don't use it in any of our builds.
> Note I am reviewing this line because the last three commits you made are all about the rewrite of our approach to testing LRA.

I cannot find any references to or documentation about this file.

Reply:
This is not a file we provide. This file is copied from `${env.JBOSS_HOME}/standalone/configuration/standalone.xml` and renamed to `lra-coordinator.xml`. We need two standalone XML files to start two separated instances of WildFly (AFAIK). `lra-coordinator.xml` and `lra-participant.xml` are created at lines 239-240 of `rts/lra/test/pom.xml`.